gensim/models/callbacks.py

```diff
@@ -578,6 +578,14 @@ class CallbackAny2Vec:
 
     See examples at the module level docstring for how to define your own callbacks by inheriting  from this class.
 
+    Important
```

Comment:
I doubt this is really important. Extremely marginal functionality, saw maybe 2 users in 4 years.

I'd suggest not mentioning it at all (beyond the Migration notes). Or at least a less alarming heading:

```suggestion
    Note on pre-4.0 compatiblity
```